Makefile

```diff
@@ -52,8 +52,8 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build . -t ${IMG}
-	docker build . -t ${WORKERIMG} --file Dockerfile.worker
+	docker build  -f ./controllers/Dockerfile . -t ${IMG}
+	docker build  -f ./worker/Dockerfile.worker . -t ${WORKERIMG}
```

Comment:
Let's rename this file to `./worker/Dockerfile` since now we have it in its own subdirectory.